tools/eqget/GEQDSK.py

```diff
@@ -50,6 +50,7 @@ def get_flux_surface(self, psi_n, theta=None, closedContourTol = 1e-6):
         #    vertices = v[-1]
             
         vertices = self.contour_generator.create_contour(psi_n)
+        vertices = vertices[0]
```

Comment:
This reminds me... This line is due to a change made in some version of matplotlib. We should have an if statement instead since this will break on older versions. I'll add this on a separate branch, so just ignore this comment for now.